build.sbt

```
@@ -0,0 +1,50 @@
+addCommandAlias("fmt", "; all root/scalafmtSbt root/scalafmtAll")
+
+val zioVersion            = "2.0.13"
+val zioJsonVersion        = "0.5.0"
+val zioConfigVersion      = "3.0.7"
+val zioLoggingVersion     = "2.1.12"
+val logbackClassicVersion = "1.4.7"
+val testContainersVersion = "0.40.15"
+val zioMockVersion        = "1.0.0-RC11"
+val zioNioVersion         = "2.0.1"
+val zioPreludeVersion     = "1.0.0-RC19"
+val zioHttpVersion        = "3.0.0-RC1"
```

```suggestion
val zioHttpVersion        = "3.0.0-RC2"
```

would probably be good to set up scala steward soon, so we get version upgrades automatically